JamGame/Assets/Scripts/Level/Sheduler.cs

```diff
@@ -15,6 +15,20 @@ public class Sheduler : MonoBehaviour
 
         private IEnumerator<DayConfig> dayEnumerator;
         private IEnumerator<IDayAction> actionEnumerator;
+        private bool isPlanned = true;
+
+        // Must be catched by Executor
+        public void ActionEndRealese()
```

Comment:
Sorry what?)

Reply:
See Executor UnityEvent for understanding this comment

Reply:
line 21, what's Realese?

Reply:
Relese

Reply:
you shure?

Reply:
Activation